tests/test_client.py

```diff
@@ -503,3 +503,49 @@ def test_list_teammates():
 #     assert len(new_teammates) >= len(
 #         old_teammates
 #     )  # needs to sleep for teammates list to be updated
+
+
+def test_get_tasks_without_project_name():
+    with pytest.raises(ValueError):
+        list(client.get_tasks())
+
+
+def test_get_tasks_with_optional_project_name():
+    batch = create_a_batch()
+    tasks = []
+    for _ in range(3):
+        tasks.append(make_a_task(batch=batch.name))
+    task_ids = {task.id for task in tasks}
+    for task in client.get_tasks(
+        project_name=None,
+        batch_name=batch.name,
+        limit=1,
+    ):
+        assert task.id in task_ids
+
+
+def test_process_tasks_endpoint_args_with_optional_project_name():
+    args = client._process_tasks_endpoint_args(
+        project_name=None, batch_name="test_batch"
```

Comment:
Should we testing by removing `project_name=None` and only leaving batch_name?

Reply:
hmm let me revise it

Reply:
I've updated the `test_process_tasks_endpoint_args_with_optional_project_name` function

Reply:
thank you!